R/DiseasystoreEcdcRespiratoryViruses.R

```diff
@@ -0,0 +1,108 @@
+#' @title feature store handler of EU-ECDC Respiratory viruses features
```

Comment:
Shouldn't just be ECDC instead of EU-ECDC?

Reply:
I would personally agree, but I took this name from their own github repo:

https://github.com/EU-ECDC/Respiratory_viruses_weekly_data

Here they call themselves "EU-ECDC" so I went with that.